### DISCUSSION TITLE
Adding Delete Functionality Patch from Release 6 to Release 5.0/5.1 to mobile app

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#706] [General Discussion]
Hello Team,

We are currently on release 5.0 and in the process of deploying 5.1. However, we are encountering issues during this deployment phase, and it may take some time to fix them. Meanwhile, we have identified a crucial delete functionality patch present in release 6 that we would like to integrate into our current release (5.0 or 5.1) to address certain Google Play policies.

I have a few questions:
1. Is it feasible to add the delete functionality patch from release 6 to either release 5.0 or 5.1?
2. If it is possible, where can I find this patch?
3. Can anyone provide guidance or assistance in this matter?

Your insights and assistance are highly appreciated.

cc: @rajeevsathish @mohitga @vinukumar-vs @ashokreddy1208 @swayangjit

**Comment by swayangjit** [comment]
@vinukumar-vs  Can you please take up this discussion?

**Comment by vinukumar-vs** [comment]
@siddhishinde0723 This feature is currently in testing phase of release-7.0.0. We are planning to give sign-off (adopters can use release-7.0.0) by April end.

Our design of delete user functionality is generic and the patch can be applied on any older version. You can find the related final document in the release notes of release-7.0.0 by Apr end.

FYR: Please find the below JIRA ticket and refer the attached design documents to the JIRA story for better understanding
https://ed.sunbird.org/v/release-7.0.0-draft/use/release/release-notes#id-1.-delete-user-functionality-lr-683



[@AmiableAnil](https://github.com/AmiableAnil) [@PoojaJManjooran](https://github.com/PoojaJManjooran) @anisha147 @mohitga @santhosh-tg  @ashokreddy1208

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

Would you like to continue this discussion further? If not, we can close this thread and initiate a new one once we release 7.0.0.

cc: @vinukumar-vs

**Reply by siddhishinde0723** [reply]
Hi @anisha147

Sure, I'll close this thread. 

Thank you, @vinukumar-vs for the details
